libs/sdk-ui-dashboard/src/model/types/commonTypes.ts

```diff
@@ -82,6 +82,14 @@ export type DashboardConfig = {
      * Defaults to false i.e. NOT a read-only mode.
      */
     isReadOnly?: boolean;
+
+    /**
+     * When dashboard is embedded via iframe, this property must be set to true.
```

Comment:
This comment does not make much sense, I don't think anyone will embed this using iframe anymore :-/

Reply:
This is mainly for KD purposes (you are still able to embed KD via iframe). I'm not sure about better way to handle it.

Reply:
Oh, right, I did not realize that.